components/tidb_query/src/codec/mysql/time/mod.rs

```diff
@@ -216,6 +216,12 @@ impl From<TimeType> for FieldTypeTp {
     }
 }
 
+impl Default for TimeType {
+    fn default() -> Self {
+        TimeType::Date
```

Comment:
Although I know it is for deriving Default for TimeArgs, but this default itself does not make sense..

Reply:
I got it. I remove the #[derive(.., Default)] and only impl Default for `TimeArgs`:
```Rust
impl Default for TimeArgs {
    fn default() -> Self {
        TimeArgs {
            year: 0,
            month: 0,
           day: 0,
           hour: 0,
           minute: 0,
           second: 0,
           micro: 0,
           fsp: 0,
           time_type: TimeType::Date,
        }
    }
}